packages/interface/README.md

```diff
@@ -70,3 +70,59 @@ wp.data.select( 'core/interface' ).isItemPinned( 'core/edit-post', 'edit-post-bl
 ```
 
 <br/><br/><p align="center"><img src="https://s.w.org/style/images/codeispoetry.png?1" alt="Code is Poetry." /></p>
+
+### Preferences
+
+The interface package provides some helpers for implementing editor preferences.
+
+#### Features
+
+Features are boolean values used for toggling specific editor features on or off.
+
+Set the default values for any features on editor initialization:
+
+```js
+import { setFeatureDefaults } from '@wordpress/interface';
+
+function initialize() {
+	// ...
+
+	setFeatureDefaults( 'namespace/editor-or-plugin-name', {
+		myFeatureName: true
+	} );
+
+	// ...
+}
+```
+
+Use the `toggleFeature` action and the `isFeatureActive` selector to toggle features within your app:
```

Comment:
I don't fully understand how the persistence plugin works. But I feel like `toggleFeature` might not be the best API for this. AFAIK, the persistence plugin will synchronize changes to `localStorage` by default, which is the same across multiple tabs. Unless we've also implemented some [broadcasting channels](https://developer.mozilla.org/en-US/docs/Web/API/Broadcast_Channel_API) to synchronize changes across different tabs, the state in UI and the data in `localStoage` can easily be out of sync, leading to unexpected bugs. Unless we only load from `localStorage` for the initial hydration? Then I think it should be fine... I guess 😅.

Reply:
I don't think that's too much of an issue. It probably will be out of sync in the case of multiple tabs, but they're simple settings that won't break if they're set to different values. The PR also doesn't introduce anything new, the settings will work the same as before.

My understanding of the way it works is that it does hydrate from `localStorage` initially. But then further changes to the store are also replicated to `localStorage`, but you're right that changes from another instance won't be pushed back into the store (from what I understand).

The eventual goal btw will be to persist these settings to the database instead of local storage - https://github.com/WordPress/gutenberg/pull/19177.

Reply:
> The eventual goal btw will be to persist these settings to the database instead of local storage 

Both issues existed from start. The lack of persisting the settings between browsers and devices is a more pressing issue 😅 

Anyway, it's out of scope of this PR that is focused on moving the functionality to `@wordpress/interface`.

Reply:
What I mean here is mostly about the `toogleFeature` API, since that we're making it graduating from `__unstable` prefix in this PR. Would it be less confusing if we make it into `setFeatureIsActive` which accepts absolute values instead?

Reply:
`toggleFeature` is only `unstable` in the widgets editor because I had the intention to replace it in the future. In the post editor it's stable, so it's good to have a matching API so that backward compatibility can be maintained.